portal-backend/depmap/public/views.py

```diff
@@ -9,6 +9,8 @@
 from depmap.public.resources import (
     get_root_category_subcategory_topics,
     create_sanitizer,
+    refresh_all_category_topics,
+    _read_forum_api_key,
```

Comment:
A bit surprising to see an import with a leading underscore. Usually that means it's supposed to be private to the module.

Reply:
We have multiple places in the portal where we import functions with a leading underscore but I agree that's not really the best practice. I had originally intended to use this function as part of public function but in the end decided not to. I'll rename this to not imply a private function though!